pool/pooler.go

```diff
@@ -1,15 +1,28 @@
 package pool
 
 import (
+	"context"
 	"time"
 
 	"github.com/tarantool/go-tarantool/v2"
 )
 
+// PoolerTopology is the interface that must be implemented by a connection pool and
+// describe topology methods.
+type PoolerTopology interface {
+	Add(ctx context.Context, instance Instance) error
+	Remove(name string) error
+}
```

Comment:
Please, let's use a more general name here and a bit more Go-like for interfaces (with `er`).

```suggestion
// TopologyEditor is the interface that must be implemented by a connection pool and
// describe edit topology methods.
type TopologyEditor interface {
	Add(ctx context.Context, instance Instance) error
	Remove(name string) error
}
```

Reply:
iv fixed it with amend